samples/tfm/tfm_hello_world/sample.yaml

```diff
@@ -21,7 +21,7 @@ common:
         - "Hashing 'Hello World! .*'"
         - "SHA256 digest:"
         - "0x[0-9a-f]{64}"
-        - "(Configuring MCU selection for LFXO)?"
+        - "MCU selection .*"
```

Comment:
It will be more future-proof to delete this line.

regexing on this does not prove anything that is not already proven by the above regex (there are no return on error
paths to verify).

CI does not fail if there is more printed than what we regex.

Reply:
Having this line does give confidence that nothing happened with execution during `gpio_pin_mcu_select` though, like if the chip halted or reset. I don't know how likely that is to ever happen, but I'd vote to keep the line.

Reply:
I didn't consider that. Fair enough.